setup.py

```diff
@@ -20,6 +20,7 @@
     author='Ryan Tilder',
     author_email="service-dev@mozilla.com",
     url="http://mozilla.org",
+    install_requires=["M2Crypto"],
```

Comment:
have you checked with ops that this is ok? They install M2Crypto from RPM so it might not install right with this.